<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistManagement</class>
 <widget class="QDialog" name="PlaylistManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_playlists">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Your Playlists</string>
   </property>
   <widget class="QListWidget" name="listWidget_playlists">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_songs">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Songs</string>
   </property>
   <widget class="QTableWidget" name="tableWidget_songs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_PMTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Playlist Management Page</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="createPlaylist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>

  </widget>
  <widget class="QPushButton" name="createPlaylist_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rename Playlist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createPlaylist_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete Playlist</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>

  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
